admin_manual/configuration_server/android_deep_link_handling.rst

```diff
@@ -0,0 +1,49 @@
+Deep Link Handling
+====================================
+
+Deep linking in Android allows your application to be launched directly from a URL, 
+making it easier for users to navigate to specific content within your app. 
+Starting from Android 12, handling deep links requires additional configuration 
+using an assetlinks.json file to ensure the app and the host domain are properly 
+associated.
+
+Deep Linking in Android 11 and Below
+====================================
+For Android 11 and below, deep linking is straightforward and does not require additional 
+configuration beyond the usual manifest settings.
+
+Deep Linking in Android 12 and Above
+====================================
+For Android 12 and above, an additional configuration step is required to verify the 
+relationship between your app and the host domain using the assetlinks.json file.
+
+Creating assetlinks.json:
+-------------------------
+Create a file named assetlinks.json and host it in the .well-known directory of 
```

Comment:
```suggestion
Create a file named ``assetlinks.json`` and host it in the .well-known directory of 
```

Does this actually work in with our standard web server configs? Our distributed `.htaccess` doesn't seem to handle creating a `.well-known` folder and placing something in it as-is (other than `acme-challenge|pkivalidation` I think).

Reply:
I don't have idea about the standard web server configs. Maybe @tobiasKaminsky can help?

Reply:
Hm. No real idea, but we also have .well-known for caldav and carddav.